docs/.vitepress/config.ts

```diff
@@ -53,6 +53,31 @@ function getSideBarWithExpandedEntry(entryToExpand: string): SidebarItem[] {
       text: 'API',
       items: apiPages,
     },
+    {
+      text: 'Contributing',
+      items: [
+        {
+          text: 'Code of Conduct',
+          link: '/contributing/code-of-conduct',
+        },
+        {
+          text: 'Report Bugs',
```

Comment:
The other headlines use the `ing` form.

```suggestion
          text: 'Reporting Bugs',
```

or switch the existing `ing`s to not-`ing`